extensions/wikia/Insights/scripts/InsightsLoopNotificationTracking.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * LoopNotificationTracking for Insights extension
+ * Module sends tracking record to google analytics and internal analytics
+ * for events related to BannerNotification appearance and clicks.
+ *
+ * checks for insights url param to log specific insight type
+ * checks for item_status url param to log action status (fixed/notfixed)
+ */
+define('ext.wikia.Insights.LoopNotificationTracking',
+	['jquery', 'wikia.tracker', 'wikia.querystring'],
+	function ($, tracker, Querystring) {
+		'use strict';
+
+		var qs = new Querystring(),
+			insightType = qs.getVal('insights', null),
+			itemStatus = qs.getVal('item_status', null);
+
+		/* If there's no itemStatus use action parameter */
+		if (itemStatus === null) {
+			itemStatus = qs.getVal('action', null);
```

Comment:
That's for identifying edit mode in logs - I'll change it to use notificationType that you prepared recently here https://github.com/Wikia/app/pull/7037/files#diff-274f0ab383736c0231df0070ba374d63R34